### PR TITLE
Update Diana README with new documentation for 0.0.6a10

### PR DESCRIPTION
**Pull request by erikabarcott**
# Description
Add new documentation for 0.0.6a10 to Diana README.

# Issues

# Other Notes
I incorporated all your changes from [your gist version](https://gist.github.com/NeonDaniel/299a4407d914b97d6ad5debc2a4618e5/d27e0513b598780e307a0e47f9cb45eeef8e768c), but I slightly reworded the Let's Encrypt bits.

### CODE REVIEW

**Comment by NeonDaniel** [comment]
Rendered doc viewable at: https://github.com/erikabarcott/neon-diana-utils/tree/new-diana-docs

File: README.md [hunk 1]
```diff
@@ -1,31 +1,207 @@
...
+* [Ubuntu](https://ubuntu.com/) 20.04 or later. 
+  * Diana will most likely work on other flavors of Linux, but we have not yet verified this.
+* [Python](https://www.python.org/getit/) 3.8 or later.
+* [Pip](https://pypi.org/project/pip/) Python package installer.
+* [Kubectl](https://kubernetes.io/docs/reference/kubectl/) Kubernetes command-line tool.
+* [Helm](https://helm.sh/) package manager for Kubernetes.
+* A [Kubernetes](https://kubernetes.io/) installation.
+  * The following instructions assume a local installation using [Microk8s](https://microk8s.io/) version 1.26/stable or later.
+  * You can likely deploy Diana on a Kubernetes cluster in the cloud, but we have not yet verified this.
+* [Neon Skill Utilities](https://github.com/NeonGeckoCom/neon-utils) 1.0 or later.
```
Comment (NeonDaniel) [inline README.md:20]: This shouldn't be necessary to install separately; was there an with with dependencies after installing just diana?

File: README.md [hunk 2]
```diff
@@ -1,31 +1,207 @@
...
+
+3. Use Pip to install Diana:
+
+```
+pip install --pre neon-diana-utils
+```
+
+This command installs the newest pre-release version, which is described in this tutorial. 
+
+**Warning:** You can use `pip install neon-diana-utils` to install the current stable version. However, that version does not include Helm chart support. For information on installing and running the current stable version, see [the archived documentation in the README file](https://github.com/NeonGeckoCom/neon-diana-utils/blob/dev/README.md).
```
Comment (NeonDaniel) [inline README.md:53]: Maybe update this note to indicate that Helm support is (will be?) added in version `0.1.0` so we don't have to edit it before release

File: README.md [hunk 3]
```diff
@@ -1,31 +1,207 @@
...
+
+For more information on the available versions of Diana, see [the Python Package Index repo for Neon Diana](https://pypi.org/project/neon-diana-utils/).
+
+4. Verify Diana is installed:
+
+```
+diana --help
```
Comment (NeonDaniel) [inline README.md:60]: Consider `diana --version` here so the version can be verified

**Review by NeonDaniel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (NeonDaniel) [reply README.md:25]: You shouldn't have to touch `neon-utils` at all; if you do then its something I should fix in `neon-diana-utils`

File: README.md [hunk 4]
```diff
@@ -1,31 +1,209 @@
...
+  make-keys-config      Generate a configuration file with access keys
+  make-rmq-config       Generate RabbitMQ definitions
+  start-backend         Start a Diana Backend
+  stop-backend          Stop a Diana Backend
+
+```
+
+## Deploy NeonAI Diana
+
+This example deployes Diana in a local [Microk8s Kubernetes](https://microk8s.io/) cluster.
```
Comment (NeonClary) [inline README.md:94]: Deployes --> deploys

**Review by NeonClary — CHANGES_REQUESTED** [review]
Two tiny proofreading changes. The content is beyond my technical capabilities, but it makes sense and seems well organized.

File: README.md [hunk 5]
```diff
@@ -1,31 +1,209 @@
...
+
+3. Use Pip to install Diana:
+
+```
+pip install --pre neon-diana-utils
+```
+
+This command installs the newest pre-release version, which is described in this tutorial. 
+
+**Warning:** You can use `pip install neon-diana-utils` to install the current stable version. Version 0.1.0 includes Helm chart support. For information on installing and running older versions, see [the archived documentation in the README file](https://github.com/NeonGeckoCom/neon-diana-utils/blob/dev/README.md).
```
[the remark this answers is not in the document]
Comment (erikabarcott) [reply README.md:54]: Exciting idea! I agree, I think it's ready for a 1.x but I'll leave it to you @NeonDaniel

File: README.md (hunk 4 above)
Comment (erikabarcott) [reply README.md]: Thank you!!

File: README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (NeonDaniel) [reply README.md:54]: Makes sense; I went ahead and cleaned up all of the deprecated methods, so we should push a 1.0.0 release next anyways (https://semver.org/spec/v2.0.0.html)

**Pull request merged**